libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <toolchain.h>
```

Comment:
I'd like to remember that using `-include` directive will require using `bld.add_manual_dependency()` (I think that's the function to use).

@lucasdemarchi I was thinking about that option a while ago... Using `-include` implies on dependency for all sources, which doesn't sound good :/